script.js

```diff
@@ -2,11 +2,164 @@
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  displayResults();
+  selectEpisode();
 }
 
-function makePageForEpisodes(episodeList) {
+function makePageForEpisodes(episodesList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+
+  let ourSection = document.createElement("section");
+  ourSection.classList.add("allEpisodes")
+  rootElem.appendChild(ourSection);
+
+  for (let episodeIndex = 0; episodeIndex < episodesList.length; episodeIndex++) {
+    let episodeContent = document.createElement("article");
+    episodeContent.classList.add("episodeWrapper");                                               //Can I make the 'class' name same as the variable name since we can not use the variable name in CSS
+
+    let episodeTitle = document.createElement("h3");
+    episodeTitle.classList.add("episodeName");
+
+    let episodeImage = document.createElement("img");
+    episodeImage.classList.add("episodePicture")
+
+    let episodeSummary = document.createElement("div");
+    episodeSummary.classList.add("episodeDescription")
+
+    episodeContent.append(episodeTitle, episodeImage, episodeSummary);
+    ourSection.append(episodeContent);
+
+    addContentToEachEpisode(episodesList, episodeIndex);
+  }
+}
+
+function addContentToEachEpisode(list, index) {
+  let title = document.getElementsByClassName("episodeName")[index];
+  title.innerHTML = `${list[index].name} - <span> ${getSeasonAndEpisodeNumber(list, index)} </span>`;
+
+  let picture = document.getElementsByClassName("episodePicture")[index];
+  picture.setAttribute("src", `${list[index].image.original}`)
+
+  let description = document.getElementsByClassName("episodeDescription")[index];
+  description.innerHTML = `${list[index].summary}`;
+}
+
+function getSeasonAndEpisodeNumber(list, index) {
+  let seasonNumber = `S${(list[index].season < 10 ? "0" : "") + list[index].season}`
```

Comment:
I'd avoid directly putting `10` here since it becomes a magic number. Create a `constant` inside the function `getSeasonAndEpisodeNumber` to store the 10 in that instead. Think of a good name for this constant that describes why `10` is special in this case. 


https://medium.com/@samaddico/programming-and-magic-numbers-f766e0cd1369

Reply:
Thanks for this valubale feedback, now I know what is a "magic number" and the benefits of avoiding them.